src/main/scala/com/redis/sentinel/SentinelMonitor.scala

```diff
@@ -111,13 +118,17 @@ class SentinelMonitor (address: SentinelAddress, listener: SentinelListener, con
 
 trait SentinelHeartBeater extends Runnable with Log{
   private var running = false
+  private val beatingSentinel = new AtomicBoolean(true) // initialised to true so we don't unnecessarily wake up ops
```

Comment:
leave it to atomic. it's fine. volatile gives happens-before guarantee. it makes reading from main memory than cpu cache. it's safer. what I meant was it's not a concurrent critical feature. anyway, let's keep it atomic. as it won't make much difference, it's better to do it right